packages/utxo-lib/tests/compose.test.ts

```diff
@@ -1,4 +1,5 @@
 import { composeTx } from '../src';
+import * as utils from '../src/compose/utils';
```

Comment:
Why not just

```
import { convertFeeRate } from '../src/compose/utils';
```

Reply:
this tests should be extended in the future to cover all the utilities in that file, not only this one

Reply:
leaving as is, it's only a test